lib/kubernetes-deploy/runner.rb

```diff
@@ -213,10 +215,22 @@ def validate_configuration
       KubernetesDeploy.logger.info("All required parameters and files are present")
     end
 
+    def update_tprs(resources)
+      resources.each do |r|
+        KubernetesDeploy.logger.info("- #{r.id}")
+        r.deploy_started = Time.now.utc
+        run_kubectl("update", "--namespace=#{@namespace}", "-f", r.file.path)
```

Comment:
Ah - I copied what `deploy_resources` was doing. d10c217 should remove the redundant `--namespace` flag in both cases.